View/src/view.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * view.h
+ *
+ *  Created on: Feb 15, 2015
+ *      Author: edem
+ */
+
+#ifndef VIEW_H_
+#define VIEW_H_
+
+#include "observer.h"
+
+/* This abstract class inherits observer's
+ * functionality and displays the data via use of
+ * the qt framework. In addition it receives
+ * commands from the user and sends them to the
+ * controller to be implemented
+ */
+
+class View : public Observer
+{
+public:
+		View();
+
+		void moveForward();
```

Comment:
These functions are not generic to a View.  These sound like commands a Controller would issue to you if they were in a View that was specific to moving the submarine.  These would not work for a scenario where the View was a Menu (you don't need to move the submarine forward in a menu)